packages/keystone/KSv5v6/v5/prepareKeystone.js

```diff
@@ -203,11 +211,22 @@ function prepareKeystone ({ onConnect, extendKeystoneConfig, extendExpressApp, s
             app.use(json({ limit: '100mb', extended: true }))
             app.use(urlencoded({ limit: '100mb', extended: true }))
 
+            // Remove \r and \n to prevent header injection attacks
+            app.use((req, res, next) => {
+                for (const header in req.headers) {
+                    if (typeof req.headers[header] === 'string') {
```

Comment:
It looks strange. What if we have the same header twice. Is the result would be list or one? Should we also drop such headers?

Reply:
Added recursive check
in some cases it will be comma-separated value, for set cookies it will be an array